templates/astropy.tpl

```diff
@@ -26,7 +26,41 @@
     </nav>
 
     {{ super() }}
-
+	
+	<!-- Edit on Github link -->
+	<div style="top:0px; border:1px">
+    <table border="1" align="center">
+     <tr height="100%">
+      <td height="100%" width="100%" valign="middle" align="center">
+	  <a href="https://github.com/astropy/astropy-tutorials/tree/master/tutorials/{abspageurl}">Edit on Github  <img src="images/gitlogo.png" height=40px width=40px/></a>
+      </td>
+     </tr>
+    </table>
+  </div>
+  
```

Comment:
I would add some (small) padding between the Edit on GitHub button and the Disqus block.